pandas/core/generic.py

```diff
@@ -1591,62 +1591,87 @@ def _repr_latex_(self):
     index_label : string or sequence, default None
         Column label for index column(s) if desired. If None is given, and
         `header` and `index` are True, then the index names are used. A
         sequence should be given if the DataFrame uses MultiIndex.
-    startrow :
-        upper left cell row to dump data frame
-    startcol :
-        upper left cell column to dump data frame
+    startrow : integer, default 0
+        Upper left cell row to dump data frame.
+    startcol : integer, default 0
+        Upper left cell column to dump data frame.
     engine : string, default None
-        write engine to use - you can also set this via the options
-        ``io.excel.xlsx.writer``, ``io.excel.xls.writer``, and
+        Write engine to use, 'openpyxl' or 'xlsxwriter'. You can also set this
+        via the options ``io.excel.xlsx.writer``, ``io.excel.xls.writer``, and
         ``io.excel.xlsm.writer``.
     merge_cells : boolean, default True
         Write MultiIndex and Hierarchical Rows as merged cells.
-    encoding: string, default None
-        encoding of the resulting excel file. Only necessary for xlwt,
+    encoding : string, default None
+        Encoding of the resulting excel file. Only necessary for xlwt,
         other writers support unicode natively.
     inf_rep : string, default 'inf'
         Representation for infinity (there is no native representation for
-        infinity in Excel)
+        infinity in Excel).
+    verbose : boolean, default True
+        Display more information in the error logs.
     freeze_panes : tuple of integer (length 2), default None
         Specifies the one-based bottommost row and rightmost column that
-        is to be frozen
+        is to be frozen.
 
-        .. versionadded:: 0.20.0
+        .. versionadded:: 0.20.0.
+
+    See Also
+    --------
+    pandas.read_excel
+
+    Examples
+    --------
+
+    >>> df1 = pd.DataFrame([['a', 'b'], ['c', 'd']],
+    ...                   index=['row 1', 'row 2'],
+    ...                   columns=['col 1', 'col 2'])
+    >>> writer = pd.ExcelWriter('output.xlsx', engine='xlsxwriter')
+    >>> df1.to_excel(writer, sheet_name='Sheet1')
+    >>> writer.save()
 
-    Notes
-    -----
     If passing an existing ExcelWriter object, then the sheet will be added
     to the existing workbook.  This can be used to save different
     DataFrames to one workbook:
 
-    >>> writer = pd.ExcelWriter('output.xlsx')
-    >>> df1.to_excel(writer,'Sheet1')
-    >>> df2.to_excel(writer,'Sheet2')
-    >>> writer.save()
+    >>> writer2 = pd.ExcelWriter('output2.xlsx', engine='xlsxwriter')
+    >>> df1.to_excel(writer2, sheet_name='Sheet1')
+    >>> df2 = df1.copy()
+    >>> df2.to_excel(writer2, sheet_name='Sheet2')
+    >>> writer2.save()
+
+    Limit floats to a fixed precision using float_format. For example
+    float_format="%.2f" will format 0.1234 to 0.12.
 
     For compatibility with to_csv, to_excel serializes lists and dicts to
```

Comment:
Can we add a link to the method `pandas.DataFrame.to_csv` here as well?

Reply:
ok